config/locales/en.yml

```diff
@@ -1387,6 +1384,44 @@ en:
     processing_request: Processing your submission, this may take some time.
     processing_samplesheet: Processing samplesheet, this may take some time.
     submission_error: Something went wrong with the submission request.
+  pathogen:
```

Comment:
Take a look at how I added translations directly into Pathogen in https://github.com/phac-nml/irida-next/pull/1123/files#diff-f72bb6ab0d53bd2f7d0f827b6131011338793380ebc2334d0346ce53f0a23781